src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -805,6 +839,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    Me.OptionExplicit = other.OptionExplicit AndAlso
                    Me.OptionCompareText = other.OptionCompareText AndAlso
                    Me.EmbedVbCoreRuntime = other.EmbedVbCoreRuntime AndAlso
+                   Me.SuppressEmbeddedDeclarations = other.SuppressEmbeddedDeclarations AndAlso
```

Comment:
It feels like Private Sub New(other As VisualBasicCompilationOptions) should be adjusted too.